authority/provisioner/gcp.go

```diff
@@ -89,6 +89,8 @@ type GCP struct {
 	ProjectIDs             []string `json:"projectIDs"`
 	DisableCustomSANs      bool     `json:"disableCustomSANs"`
 	DisableTrustOnFirstUse bool     `json:"disableTrustOnFirstUse"`
+	EnableSSHCAUser        bool     `json:"enableSSHCAUser"`
+	DisableSSHCAHost       bool     `json:"disableSSHCAHost"`
```

Comment:
maybe these should both be `Disable`-prefixed, or both be `Enable`-prefixed? or the smallstep team might prefer not having the host CA be an option.

Reply:
@dopey  How do y'all feel about this?

Reply:
I would prefer `Disable*` but at the same time as it is backward compatible. We can also achieve backward compatibility with `*bool`. assuming nil true in some cases and false in others, if we use *bool we might also want the `omitempty`. But this last option is ugly.